docs/orchestration.md

```diff
@@ -278,6 +277,78 @@ Even though **stack-c** defined that it needs to be run after **stack-b**, since
 **runtime** order.
 
 
+## Execution Environment
+
+The environment in which stacks are going to be executed can be configured
+using the `terramate.config.run.env` block. This block has no labels and
+accepts arbitrary attributes where each attribute represents an environment
+variable that will be evaluated and exported when executing the stack.
+The attributes must always evaluate to strings.
+
+Example:
+
+```hcl
+terramate {
+  config {
+    run {
+      env {
+        TF_PLUGIN_CACHE_DIR = "/some/path/etc"
+      }
+    }
+  }
+}
+```
+
+Will export the environment variable `TF_PLUGIN_CACHE_DIR` on the stack
+execution environment with the value `/some/path/etc`.
+
+Inside the `terramate.config.run.env` block the `env` namespace will be
+available including environment variables exported from the host:
+
+```hcl
+terramate {
+  config {
+    run {
+      env {
+        TF_PLUGIN_CACHE_DIR = "${env.HOME}/.terraform-cache-dir"
+      }
+    }
+  }
+}
+```
+
+Globals and Metadata can also be referenced:
+
+```hcl
+terramate {
+  config {
+    run {
+      env {
+        TF_PLUGIN_CACHE_DIR = "${terramate.stack.path.absolute}/${global.cache_dir}"
```

Comment:
idea: what if `terramate` caches the stuff by default to a path maybe specific for the terramate setup and if this environment variable set then it overrides it, and to disable it there could be something like `TF_PLUGIN_CACHE=false`?

Reply:
I think it makes sense to have a default if this is what most people need but I don't have much experience to judge the benefit of it..